### PR TITLE
[FIX] making the cleanRoomHistory method delete the files and attachments when deleting the messages

### LINKED ISSUE #10870: API cleanRoomHistory Does Not Remove Attached Media
The api which uses the `cleanRoomHistory` method does not clean up the attached files/media.

### PR DESCRIPTION
**Pull request by MarcosSpessatto**
Closes #10870

### CODE REVIEW

File: packages/rocketchat-lib/server/methods/cleanRoomHistory.js [hunk 1]
```diff
@@ -21,6 +21,13 @@ Meteor.methods({
 					$lte: latest
 				}
 			});
+			RocketChat.models.Uploads.remove({
+				rid: roomId,
+				uploadedAt: {
+					$gte: oldest,
```
Comment (ggazzo) [inline packages/rocketchat-lib/server/methods/cleanRoomHistory.js]: @MarcosSpessatto what do you think about use
```
cont gt = inclusive ? '$gte' : 'gt'

**Comment by vynmera** [comment]
How will this impact the performance of this method? On a large team with many messages, clearing a lot of them takes very very long, so long it causes a gateway timeout...

**Comment by rodrigok** [comment]
@MarcosSpessatto this does not remove the attachments, just the records that register the attachments location/url and metadata, wouldn't be better to delete the delete the files too?

**Comment by vynmera** [comment]
I've implemented further changes to this method in #11236. However, merging this may be useful for the time being until that PR is ready to merge.

As for deleting the files fully, we can use `FileUpload.getStore('Uploads').deleteById(message.file._id);`

**Comment by MarcosSpessatto** [comment]
@rodrigok This great work by @vynmera closes this issue. :100:

**Pull request closed without merging**